spec/unit/log_spec.rb

```diff
@@ -200,4 +200,16 @@
       end
     }.to raise_error(TTY::Logger::Error, "Already defined log type :success")
   end
+
+  it "logs streaming output" do
+    logger = TTY::Logger.new(output: output) do |config|
+      config.level = :debug
```

Comment:
Do we need this anymore?

Reply:
Nope! I've cleaned it up along with the rebase.